proto/viam/app/packages/v1/packages.proto

```diff
@@ -103,10 +105,12 @@ message GetPackageResponse {
 
 message ListPackagesRequest {
   string organization_id = 1;
-  optional string name = 2;
-  optional string version = 3;
-  optional PackageType type = 4;
-  optional bool include_url = 5;
+  string name = 6;
+  string version = 7;
+  PackageType type = 8;
+  bool include_url = 9;
+
+  reserved 2 to 5;
```

Comment:
Reserving the field numbers but not the names; this will break code that hasn't pulled the latest proto and using the JSON representation.

No users are using this API directly, and usage is fixed in the corresponding app and RDK PRs.